src/TimeTracker.Data/Models/TimeEntryTypeEnum.cs

```diff
@@ -1,13 +1,34 @@
+using System.ComponentModel;
+using System.Linq;
+
 namespace TimeTracker.Data.Models
 {
     /// <summary>
     /// used to describe a time entry into groups. This would be used to mark entry as Sick or Vacation time for example.
     /// </summary>
     public enum TimeEntryTypeEnum
     {
+        [Description("billable")]
         BillableProject = 0,
+        
+        [Description("nonbill")]
         NonBillable = 1,
+        
+        [Description("sick")]
         Sick = 5,
+        
+        [Description("vacation")]
         Vacation = 6
     }
+
+    public static class EnumDescription
```

Comment:
good trick of using an extension method to get around c# not letting you add methods to an enum.